docs/APP.md

```diff
@@ -1,4 +1,4 @@
-# App
+# aragonAPI for Javascript
```

Comment:
Not sure about this name, still a bit confuse about the extent of Javascript implementation of `aragonAPI`.

Reply:
Perhaps aragonAPI is enough. You've already mentioned in the first page that it's the one of the reference implementations
https://github.com/aragon/hack/pull/127/files#diff-869e39af999dedb4045ca0a0c54e727aR14